packaging/nnstreamer.spec

```diff
@@ -31,7 +31,7 @@
 %define		grpc_support 1
 %define		pytorch_support 0
 %define		caffe2_support 0
-%define		mqtt_support 0
+%define		mqtt_support 1
```

Comment:
plz consider VD / DA support (line 85 and 103)

Reply:
Thanks. It is disabled for the DA and TV profiles.